orca/orca-bakery/orca-bakery.gradle

```
@@ -34,12 +34,20 @@ dependencies {
   annotationProcessor("org.projectlombok:lombok")
 
   testImplementation(project(":orca-test-groovy"))
+  testImplementation(project(":orca-test"))
   testImplementation("com.github.tomakehurst:wiremock-jre8-standalone")
   testImplementation("org.junit.jupiter:junit-jupiter-api")
   testImplementation("org.assertj:assertj-core")
   testImplementation("org.mockito:mockito-core")
-
+  testImplementation("com.squareup.retrofit2:retrofit-mock")
+  testImplementation("com.squareup.retrofit2:converter-jackson")
+  testImplementation ("org.springframework:spring-test")
+  testImplementation("org.springframework.boot:spring-boot-starter-test")
+  testImplementation("io.spinnaker.kork:kork-retrofit2")
   testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine")
+  testImplementation("org.spockframework:spock-core")
+  testImplementation("org.spockframework:spock-spring")
```

The tests pass for me with this as testRuntimeOnly
```suggestion
  testRuntimeOnly("org.spockframework:spock-spring")
```
And the tests also pass without explicitly specifying spock-core as a (testRuntimeOnly) dependency.  spock-spring brings it in...can't decide whether it's more correct to specify spock-core or leave it out.  Tempted to leave it out.